crates/ln-dlc-node/src/ldk_node_wallet.rs

```diff
@@ -25,32 +25,32 @@ use std::sync::Arc;
 use std::time::Instant;
 use tokio::sync::RwLock;
 
-pub struct Wallet<D>
+pub struct Wallet<D, B, F>
```

Comment:
🙃 Since we only need the generics for testing, I would isolate the rest of the modules from them by providing a `pub type` which is `Wallet<sled::Tree, EsploraBlockchain, FeeRateEstimator>`.

I can do this afterwards though.

Reply:
Thank you, I'l leave it for now then.